axon-framework/messaging-concepts/supported-parameters-annotated-handlers.md

```diff
@@ -37,9 +37,13 @@ By default, `@CommandHandler` annotated methods allow the following parameter ty
 
   For more specifics on this it is recommend to read [this](message-intercepting.md#commandhandlerinterceptor-annotation) section.
 
-* If the application is run in a Spring environment, any Spring Bean can be resolved.
+* If the application is run in a Spring environment, any Spring Bean can be resolved. 
+  
+> If Spring beans are resolved for message handling methods `@Autowired` annotation should be used to get the revamped solution. 
+> 
+> The `@Qualifier` annotation can be used in conjunction with this to further specify which Bean should be resolved.
```

Comment:
This part does not have to be a comment (through `>`), but can be a part of the Spring-bulletpoint.

Reply:
Added, this piece should also be present on line 71 (about Query Handlers), since this Spring specific occurs for all types of messages.

Reply:
I think this is a better place. Let me know if you're good with it.